docs/adr/ADR-021-vm-exclusion-rules-persistence.md

```
@@ -0,0 +1,37 @@
+# ADR-021: VM exclusion rules persist, raw rows do not
+
+## Status
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)